python/looker_sdk/error.py

```diff
@@ -21,33 +21,128 @@
 # THE SOFTWARE.
 
 import attr
-from typing  import Optional, Sequence
+from typing import cast, Dict, Optional, Sequence, Tuple
+import requests
+import json
+import re
 
 """API error class
 """
 
+
 @attr.s(auto_attribs=True, kw_only=True)
-class ErrorDetail():
+class ErrorDetail:
     """Error detail:
-        documentation_url: documentation link
-        field: field with error
-        code: error code
-        message: error info message
+    documentation_url: documentation link
+    field: field with error
+    code: error code
+    message: error info message
+    error_doc_url: URL that may point to additional useful information
+    error_doc: Markdown doc that may contain additional useful information
```

Comment:
This is a nice addition to the error class, but I'm wondering if `error_doc` retrieval could cause cascading "SDK" failures if access to the CDN is blocked for the SDK's running location

Reply:
I catch all request errors at line 147-148 and jsut put empty strings in these fields if anything goes wrong.